pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -135,6 +138,38 @@ func (mi *StorageInitializerInjector) InjectStorageInitializer(pod *v1.Pod) erro
 		}
 		podVolumes = append(podVolumes, pvcSourceVolume)
 
+		// check if using direct volume mount to mount the pvc
+		// if yes, mount the pvc to model local mount path and return
+		if mi.config.PVCMountOption == PvcMountViaVolumeMount {
```

Comment:
With this option, we no longer need to inject the storage initializer container right?

Reply:
nvm, I missed the `return nil` part